go-fuzz/main.go

```diff
@@ -94,3 +98,16 @@ func main() {
 
 	select {}
 }
+
+// expandHomeDir expands the tilde sign and replaces it
+// with current users home directory.
+func expandHomeDir() {
+	usr, _ := user.Current()
+	if len(*flagWorkdir) > 2 && (*flagWorkdir)[:2] == "~/" {
```

Comment:
Please do this as:
```
*flagWorkdir = expandHomeDir(*flagWorkdir)
*flagBin = expandHomeDir(*flagBin)
```

This will slightly reduce duplication (esp if we add e.g. os.ExpandEnv in future).